forge/Installation and Upgrading/installubuntuapachemysql.md

```diff
@@ -129,8 +106,10 @@ You can save the file now. Some explanations about this Virtual Host configurati
 
 Then you need to activate the site and reload the apache2 configuration
 
-    sudo a2ensite taoplatform
-    sudo service apache2 reload
+```shell
+sudo a2ensite taoplatform
```

Comment:
did not you need to do
_sudo a2ensite taoplatform.conf_ rather than just _taoplatform_
?

Please investigate